### DISCUSSION TITLE
does this work while using Android auto wirelessly

### DISCUSSION THREAD

**Discussion by NextCodeLLC** [agronick/aa-torque#16] [Q&A]
I've done all the install steps on my Pixel 7 pro, running Android 14 and it's not showing up in Android Auto. Auto. I am using Android auto wirelessly with the Motorola dongles so I'm not sure if that's a reason why or not?

**Comment by KVance140** [comment]
Wireless and wired it's not showing up. 
22 Honda Passport
Android 14, Pixel 6
AA Torque: 10.0.23
Android Auto: 10.4.633624-release

Shows installed in AA package list

**Comment by agronick** [comment]
Pixel phones have specific provisions to prevent the Kingstaller hack from working. Moreso if you have Android 14. You will probably need root and maybe something like this. https://github.com/Rikj000/Android-Auto-XLauncher-Unlocked

**Comment by DRB523** [comment]
I have it working via wireless on my Google Pixel 4 with Android 13.  I didn't have to use Kingstaller; just had to give permissions. I am having a different issue [here](https://github.com/agronick/aa-torque/discussions/106) with the data feed to AA without initiating on my phone first  though. Any thoughts are appreciated.

**Reply by aliatx2017** [reply]
I have it working on Pixel 7/Android 14, no root - with AA Wireless adapter though (developer settings have to be enabled in AA Wireless app).
